include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
FYI, it's not possible for "long long" to be <8 bytes.  Both the C and C++ standards require that  "long long" must be equal to or (in some far-flung future) greater than 8 bytes long.

So, while it might be useful as documentation for this check to be there, there's no architecture existing today that I'm aware of where it will be false (it would require a 128bit computer or something... and even then it's not entirely clear that the compiler implementers on those systems will choose to make "long long" be 128bits).

https://en.wikipedia.org/wiki/C_data_types
http://en.cppreference.com/w/cpp/language/types